### PR TITLE
[maketmpl] parsing images in markdown files.

### PR DESCRIPTION
**Pull request by aslonnie**
for semi-reliably parsing and generating embedded img from
readme files

### CODE REVIEW

**Comment by aslonnie** [comment]
cc @can-anyscale

**Comment by khluu** [comment]
I don't know much about HTML so it's better to leave this for someone who knows more frontend to review!

File: ci/maketmpl/md_image.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+	src string
+	alt string
+
+	heightPx string
+	widthPx  string
+
+	style string
+
+	isHTML bool
```
Comment (lanbochen-anyscale) [inline ci/maketmpl/md_image.go:23]: seems `isHTML` always true? can be removed?

Comment (aslonnie) [reply ci/maketmpl/md_image.go:23]: it is used later. non-html / md image tags are directly parsable with regex, so it is much simpler.. but let me add it in here. give me a minute.

Comment (aslonnie) [reply ci/maketmpl/md_image.go:26]: added the part that is parsing images from entire md file now.

Comment (aslonnie) [reply ci/maketmpl/md_image.go:26]: and now the field is used :)

**Review by lanbochen-anyscale — APPROVED** [review]

**Pull request merged**